setup.py

```diff
@@ -48,7 +48,7 @@
         "shapely @ git+https://github.com/shapely/shapely.git",
         "tifftools",
         "tifffile>=2022.5.4",
-        "pyvips",
+        "pyvips>=2.2.1",
         "tqdm",
         "pillow",
```

Comment:
I think for the tiffwriter to work, we need to specify Pillow version to be 9.2.0. The class `PIL.Image.Resampling` doesn't exist in 9.0.0 and lower version of PIL.